Automattic/jetpack
# Instant Search: ensure the hidden overlay doesn't add whitespace to printed pages

Fixes https://github.com/Automattic/jpop-issues/issues/7134.

Note: this PR is branched off https://github.com/Automattic/jetpack/pull/21637.

The only confirmed theme that this has happened on is 'Field Guide' but there may be others.

#### Changes proposed in this Pull Request:
Ensure the search overlay is completely hidden on printed pages.

On certain themes, the overlay's presence can cause blank pages to be added to the printout.

#### Does this pull request change what data or activity we track or use?
No.

#### Testing instructions:
1. Open a site with Instant Search installed. Don't start a search.
2. Bring up the print preview for the page.
3. Ensure there are no extra blank pages added by the hidden overlay.
4. Inspect the print styles in the browser and ensure that 'display:none' is applied to '.jetpack-instant-search__overlay'.
